docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.android.mdx

```diff
@@ -0,0 +1,21 @@
+The quickest way to get started is by wrapping your App component (written in Jetpack Compose) with the Authenticator widget.
+Once an end-user has created an account & signed in, your content will be displayed.
```

Comment:
Same nit, can you combine these to one paragraph, or put new line between the two?